moochub-schema.json

```diff
@@ -1,555 +1,3105 @@
+                    },
+                    "type": {
+                      "type": "string",
+                      "description": "The type of the object.",
                       "enum": [
-                        "Person",
                         "Organization"
-                      ],
-                      "description": "Type of the instructor to specify whether this is a \"Person\" or an \"Organization\""
+                      ]
                     },
-                    "role": {
+                    "description": {
                       "type": [
                         "string",
                         "null"
                       ],
-                      "description": "Course role of the instructor"
+                      "description": "A short description of the organization. Should be provided in HTML.",
+                      "example": "openHPI is a digital educational platform situated at the HPI with a focus on topics regarding computational sciences, internet and world wide web and design thinking."
                     },
                     "image": {
-                      "title": "Visual",
-                      "type": [
-                        "object",
-                        "null"
-                      ],
-                      "description": "Details of the visual (image / video depending on the context)",
-                      "required": [
-                        "url",
-                        "licenses"
-                      ],
+                      "title": "ImageObject",
+                      "type": "object",
+                      "description": "This schema describes the attributes of an image.",
                       "properties": {
-                        "url": {
+                        "description": {
+                          "type": "string",
+                          "description": "A short description of the image. Can be used as an alternative text, if the image cannot be displayed.",
+                          "example": "This is an image. It shows ..."
+                        },
+                        "type": {
+                          "type": "string",
+                          "description": "The type of the object.",
+                          "enum": [
+                            "ImageObject"
+                          ]
+                        },
+                        "contentUrl": {
                           "type": "string",
-                          "description": "Location of the visual",
-                          "format": "iri"
+                          "format": "iri",
+                          "description": "The IRI pointing at the actual image.",
+                          "example": "https://openhpi.azureedge.net/assets/defaults/course-bb5a40d888217b3563ca4da5d868e770f43bb9ed70f1619147af98162b86ec3e.png"
                         },
-                        "licenses": {
+                        "license": {
```

Comment:
Why singular naming if this is an array? 🤔

Reply:
This is again based on the vocabulary of the underlying standards schema.org and AMB.